Assets/Scripts/Augment/BulletModifiers/KnockbackOnShotModifier.cs

```diff
@@ -33,7 +36,10 @@ public void Detach(ProjectileController projectile)
     private void KnockAwayTargets(HitboxController controller, ref ProjectileState state)
     {
         Vector3 normal = gunController.transform.forward;
-
+        if (collidersHitWithShot.Item1 == state && collidersHitWithShot.Item2.Contains(controller))
```

Comment:
note that you can actually name the fields in a tuple like `(Foo foo, Bar bar)`

Reply:
Also I'm not sure why this isn't just two variables?

Reply:
Could have been, but I think they should be a tupple as they are only meant to be used together